### PR TITLE
TeamSonarr.Sonarr version 3.0.6.1196

### PR DESCRIPTION
**Pull request by seansusmilch**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/14438)

### CODE REVIEW

File: manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: TeamSonarr.Sonarr
+PackageVersion: 3.0.6.1196
+Installers:
+- Architecture: neutral
+  InstallerType: inno
+  InstallerUrl: https://services.sonarr.tv/v1/download/main/latest?version=3&os=windows&installer=true
```
Comment (OfficialEsco) [inline manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.installer.yaml:8]:
```suggestion
  InstallerUrl: https://download.sonarr.tv/v3/main/3.0.6.1196/Sonarr.main.3.0.6.1196.windows.exe
```

**Comment by denelon** [comment]
@OfficialEsco we will hold off on merging this until you or another of our future moderators approves this PR.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Several changes need to be done.

File: manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: TeamSonarr.Sonarr
+PackageVersion: 3.0.6.1196
+Installers:
+- Architecture: neutral
```
Comment (ItzLevvie) [inline manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.installer.yaml:6]:
```suggestion
- Architecture: x86
```

File: manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: TeamSonarr.Sonarr
+PackageVersion: 3.0.6.1196
```
Comment (ItzLevvie) [inline manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.installer.yaml:4]:
```suggestion
PackageVersion: 3.0
```
image

File: manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: TeamSonarr.Sonarr
+PackageVersion: 3.0.6.1196
+PackageLocale: en-US
+Publisher: Team Sonarr
+PackageName: Sonarr
+License: Creative Commons 3.0 License
+ShortDescription: Sonarr Setup
```
Comment (ItzLevvie) [inline manifests/t/TeamSonarr/Sonarr/3.0.6.1196/TeamSonarr.Sonarr.locale.en-US.yaml:9]:
```suggestion
ShortDescription: Sonarr is a PVR for Usenet and BitTorrent users.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by OfficialEsco** [comment]
` Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\t\TeamSonarr\Sonarr\3.0.6.1196', expected 'manifests\t\TeamSonarr\Sonarr\3.0'`

**Pull request closed without merging**